Eastern-Research-Group/mywaterway
# feature/hmw-3_change-widget

## Related Issues:
* https://jira.epa.gov/browse/HMW-3

## Main Changes:
* Updated the info icon to use a darker image.
* Added a tooltip to the info icon
* Removed unused background images

## Steps To Test:
1. Navigate to http://localhost:9090/widgets/community-widgets.html
2. Verify the info icon is darker than the one at https://mywaterway-dev.app.cloud.gov/widgets/community-widgets.html
3. Mouse over the info icon
4. Verify the `learn more about this tool` tooltip is displayed
